--- atk-2.8.0/atk/Makefile.am.orig	2014-03-25 15:09:30.541269668 +0100
+++ atk-2.8.0/atk/Makefile.am	2014-03-25 15:10:26.001268029 +0100
@@ -188,6 +188,7 @@
 libatk_1_0_la_DEPENDENCIES = atk-win32-res.o atk.def
 
 install-def-file:
+	$(MKDIR_P) $(DESTDIR)$(libdir)
 	$(INSTALL) atk.def $(DESTDIR)$(libdir)/atk-1.0.def
 uninstall-def-file:
 	-rm $(DESTDIR)$(libdir)/atk-1.0.def
@@ -213,7 +214,7 @@
 endif
 
 atk.def: atk.symbols
-	(echo -e EXPORTS; $(CPP) -P -DINCLUDE_VARIABLES -DG_OS_WIN32 -DALL_FILES - <$(srcdir)/atk.symbols | sed -e '/^$$/d' -e 's/^/	/' -e 's/G_GNUC_[^ ]*//g') > atk.def
+	(echo EXPORTS; $(CPP) -P -DINCLUDE_VARIABLES -DG_OS_WIN32 -DALL_FILES - <$(srcdir)/atk.symbols | sed -e '/^$$/d' -e 's/^/	/' -e 's/G_GNUC_[^ ]*//g') > atk.def
 
 atk-$(ATK_API_VERSION).lib: libatk-$(ATK_API_VERSION).la atk.def
 	lib -machine:$(LIB_EXE_MACHINE_FLAG) -name:libatk-$(ATK_API_VERSION)-$(LT_CURRENT_MINUS_AGE).dll -def:atk.def -out:$@
